src/main/java/org/spongepowered/common/data/provider/entity/EntityDataProviders.java

```diff
@@ -592,22 +592,22 @@ private void registerFox() {
     }
 
     private void registerPanda() {
-//        register(PandaEntity.class, Keys.EATING_TIME, getter, setter); // TODO
+        register(new PandaEntityEatingTimeProvider());
```

Comment:
It's possible to use `PandaEntityAccessor` as a target type, so you can just use method references.

Reply:
Thanks, I'll know